Cineaste/ViewController/SearchMovies/SearchMoviesViewController+UITableView.swift

```diff
@@ -24,6 +24,23 @@ extension SearchMoviesViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell: SearchMoviesCell = tableView.dequeueCell(identifier: SearchMoviesCell.identifier)
 
+        // last cell for current page
+        if indexPath.row == movies.count - 1,
+            let total = totalResults {
+
+            if total > movies.count {
+                loadRecent { movies in
```

Comment:
can't this possibly be called mutiple times when I'm scrolling up and down?
I would think of implementing https://developer.apple.com/documentation/uikit/uitableviewdatasourceprefetching/1771764-tableview
instead.

Reply:
That's true, I moved the code to the `tableView(_ tableView: UITableView, prefetchRowsAt indexPaths: [IndexPath])` to trigger the network request earlier and I added a boolean to indicate if the request is still running and not to start a new request.